IdentityCore/src/requests/broker/MSIDSSOExtensionGetAccountsRequest.m

```diff
@@ -115,14 +115,15 @@ - (void)executeRequestWithCompletion:(nonnull MSIDGetAccountsRequestCompletionBl
     NSError *error;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->